### PR TITLE
update README with example config

### PR DESCRIPTION
**Pull request by madcoda**
Updated README, I think this is clearer. BTW, I suggest adding a License.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,75 @@
...
-## Start mysql database (if you use MySQL)
-
-change the MySQL connection settings in the config file:
+### Config database 
+change the MySQL connection settings in the config file: `gov_track_hk_web/gov_track_hk_web/settings.py`
 
+Example config (MySQL)
 ```
-gov_track_hk_web/gov_track_hk_web/settings.py
+DATABASES = {
```
Comment (howawong) [inline README.md]: I don't recommend to type password or any credential in the settings.py (Even in development)

I would stick to this
```
DATABASES = {
    #'default': {
    #    'ENGINE': 'django.db.backends.sqlite3',
    #    'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
    #},
    'default': {
        'ENGINE': 'django.db.backends.mysql',
        'OPTIONS': {
            'read_default_file': '/etc/mysql/gov_track_hk.cnf',
        },
    }
}

```

And just as contributor to change the path of read_default_file

```
[client]
host = HOST
database = govlabhk
user = USER
password = PASSWORD
default-character-set = utf8
```

**Comment by madcoda** [comment]
Ok, That example is just for people to get started, since some people may not know what's inside gov_track_hk.cnf.

Actually I have already implemented the Dotenv version to read a .env file which is more common and versatile than the .cnf, And next step to dockerize it (I saw someone already did).

The settings.py will look something like this (can add more options):
```
'default': {
        'ENGINE': 'django.db.backends.mysql',
        'NAME': os.environ.get("DATABASE_NAME"),
        'USER': os.environ.get("DATABASE_USER"),
        'PASSWORD': os.environ.get("DATABASE_PASSWORD"),
        'HOST': os.environ.get("DATABASE_HOST"),
        'PORT': os.environ.get("DATABASE_PORT"),
    }
```

See if you like the idea of using environment variables

**Comment by madcoda** [comment]
Anyway, I resolved the conflict and modify the README's MySQL part base on your current settings.py
Please have a look, thanks.

**Comment by howawong** [comment]
I had conservation on environment variable. Any advantage on environment variable over my.cnf?

**Comment by madcoda** [comment]
actually this pull request has nothing to do with that, sorry for confusion
maybe we discuss it on another thread, thanks

**Comment by howawong** [comment]
Hmmmm, seems env is more convenient. I will close this PR and update `settings.py`.

**Pull request closed without merging**